Application/Features/WordFeatures/Queries/QueriesHandlersBaseClasses/QueriesWithUserWordHandlerBaseClass.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Application.Features.WordFeatures.Queries.QueryExtensions;
+using Application.Interfaces;
+using Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using MongoDB.Driver;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Application.Features.WordFeatures.Queries.QueriesHandlersBaseClasses
+{
+    public class QueriesWithUserWordHandlerBaseClass : QueriesHandlerBaseClass
+    {
+        public QueriesWithUserWordHandlerBaseClass(IWordsMongoDb context) : base(context)
+        {
+        }
+        public async Task<WordCount> UserWord(string email)
```

Comment:
Add cancellationToken

Reply:
There is no such method now. It was reworked to Handle method in the GetUserWordQuery. Async and CancellationToken were added there succesfully.